### ISSUE TITLE
Unable to run a simple Hello World program - Lesson 01

### ISSUE THREAD

**Issue by PikBot** [jpkrohling/katacoda-scenarios#9]
I get the following error when I execute the first statement: 
![screen shot 2018-04-15 at 00 22 36](https://user-images.githubusercontent.com/35295607/38771573-0c0211dc-4043-11e8-9e8f-b8a740c20b87.png)

What could be the reason for this?

**Comment by BenHall** [comment]
This is something I broken and waiting on #8  to fix it. 

In the meantime, I've manually updated the content on Katacoda.

**Comment by jpkrohling** [comment]
> In the meantime, I've manually updated the content on Katacoda.

I was wondering if it was an intermittent issue, as it was working this morning :smile:

**Comment by jpkrohling** [comment]
Closing as duplicate of #7
